packages/web-app/app/_server/sales.ts

```diff
@@ -15,11 +16,35 @@ const contract = getContract({
   client,
 });
 
+const projectStatus = async () => {
+  const start = await contract.read.start();
+  const end = await contract.read.end();
+  const current = Date.now();
+
+  if (current > end) {
+    return 'completed';
+  }
+
+  if (current < start) {
+    return 'upcoming';
+  }
+
+  return 'live';
+};
+
 export const saleDetails = async () => {
   try {
+    const headersList = headers();
+    const host = headersList.get('host');
+
     return [
       {
         project: 'Citizend',
+        status: await projectStatus(),
+        url: `${host}/projects/citizend`,
+        logo: `${host}/citizend-logo-circle.svg`,
```

Comment:
This is not the logo used in that card. I'm using an svg component `CitizenBlackBackgroundLogo`. It's easier if you just copy if from there and paste it in the public folder, so you can do this.